examples/main.tf

```diff
@@ -8,6 +8,12 @@ resource "buildkite_team" "test" {
   default_member_role = "MEMBER"
 }
 
+resource "buildkite_team_member" "member1" {
+  role = "MEMBER"
+	team_id = "VXfhnVUS78HavgtP55WhWGzT401guK38Vm9LMMeCgQD124m8xaKBRq0Fth=="
+	user_id = "VXNbwSA9hwVPpMgUXu1dWIDWf45ZwU6J7deETygiLUrKBg2TZBxuDr6aKj=="
+}
+
```

Comment:
So, this is fine, but I think going forward we could create both a team and a user and then dynamically add in the values rather than hardcode.

Reply:
Fair - honestly these values are length 58 sequences for examples only in truth 👍